backend/cms/views/events/forms/event_translation_form.py

```diff
@@ -71,4 +75,7 @@ def save(self, event=None, language=None, user=None):
         return event_translation
 
     def clean_slug(self):
-        return generate_unique_slug(self, 'event')
+        unique_slug = generate_unique_slug(self, 'event')
+        self.data = self.data.copy()
```

Comment:
Why is this copy necessary?
It only overwrites the previous data with the same data again, right?

Reply:
In Django, the form data is an immutable dict. So in order to manipulate it, I had to copy it into a mutable dict. And I had to modify it in the first place, because when the form is saved successfully, the form is rendered again and then the data is not updated, so the form contains the old, "un-cleaned" slug as placeholder, even though the clean slug was successfully saved to the database. I'm not sure if this explanation is understandable :slightly_smiling_face: 
The other forms didn't need this adaption, because they never render the form after a submission, but redirect instead.

Reply:
Ok, I just didn't notice that it is immutable. Then of course it makes sense to copy it in order to make it mutable!